src/components/Root/index.tsx

```diff
@@ -7,17 +7,20 @@ import App from 'src/components/App'
 import GlobalErrorBoundary from 'src/components/GlobalErrorBoundary'
 import AppRoutes from 'src/routes'
 import { store } from 'src/store'
-import { history } from 'src/routes/routes'
+import { getNetworkRootRoutes, history, ROOT_ROUTE } from 'src/routes/routes'
 import theme from 'src/theme/mui'
 import { wrapInSuspense } from 'src/utils/wrapInSuspense'
 import Providers from '../Providers'
-import './index.module.scss'
-import './OnboardCustom.module.scss'
-import './KeystoneCustom.module.scss'
 import StoreMigrator from 'src/components/StoreMigrator'
 import LegacyRouteRedirection from './LegacyRouteRedirection'
 import { logError, Errors, CodedException } from 'src/logic/exceptions/CodedException'
 import { loadChains } from 'src/config/cache/chains'
+import { isValidChainId, _getChainId } from 'src/config'
+import { DEFAULT_CHAIN_ID } from 'src/utils/constants'
+
+import './index.module.scss'
+import './OnboardCustom.module.scss'
+import './KeystoneCustom.module.scss'
```

Comment:
Moving them will change the cascading order. I would put them back as it's not necessary for this bug fix.